content/v1.13/concepts/patch-and-transform.md

```diff
@@ -1571,10 +1572,11 @@ patches:
 
 #### Trim suffix
 
-The {{<hover label="typeRegex" line="8">}}type: TrimSuffix{{</hover>}} removes
-the matching string and all proceeding characters. 
+The {{<hover label="typeTrimS" line="8">}}type: TrimSuffix{{</hover>}} uses 
+Go's [[TrimSuffix](https://pkg.go.dev/strings#TrimSuffix)](https://pkg.go.dev/strings#TrimSuffix) and removes characters 
```

Comment:
```suggestion
Go's [TrimSuffix](https://pkg.go.dev/strings#TrimSuffix) and removes characters
```

this formatting is not like the others and is causing the URL to be displayed on the page. This is the only instance where this is an issue.

<img width="912" alt="Screenshot 2024-04-11 at 10 28 01 AM" src="https://github.com/crossplane/docs/assets/4313439/ccc72b18-fa12-43d3-a732-678499c5fb33">

Reply:
fixed!